user-interface/src/case-detail/CaseDetailScreen.tsx

```diff
@@ -212,10 +209,10 @@ export default function CaseDetailScreen(props: CaseDetailProps) {
         const response = data as Chapter15CaseDetailsResponseData;
         setCaseBasicInfo(response.body?.caseDetails);
       })
-      .then(() => {
-        AttorneysApi.getAttorneys().then((attorneys) => {
-          setAttorneysList(attorneys);
-        });
+      .catch((_error) => {
+        // TODO: handle this error. How to we propogate to the global alert??
```

Comment:
Still needs adressed

Reply:
Yeah we probably should figure this out, but not sure if it should be done on this ticket.

Reply:
Sending the message to the global alert is very simple now.